examples/notebooks/healthcare/beam_nlp.ipynb

```diff
@@ -0,0 +1,642 @@
+        "\n",
+        "For details about Apache Beam pipelines, including PTransforms and PCollections, visit the [Beam Programming Guide](https://beam.apache.org/documentation/programming-guide/).\n",
+        "\n",
+        "You'll be able to use this notebook to explore the data in each PCollection."
+      ],
+      "metadata": {
+        "id": "nEUAYCTx4Ijj"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "First, lets install Apache Beam."
+      ],
+      "metadata": {
+        "id": "ZLBB0PTG5CHw"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "pip install apache-beam[gcp]"
+      ],
+      "metadata": {
+        "id": "O7hq2sse8K4u"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Set the variables in the next cell based upon your project and preferences. The files referred to in this notebook nlpsample*.csv are in the format with one\n",
+        "blurb of clinical note.\n",
+        "\n",
+        "Note that below, **us-central1** is hardcoded as the location. This is because of the limited number of [locations](https://cloud.google.com/healthcare-api/docs/how-tos/nlp) the API currently supports."
+      ],
+      "metadata": {
+        "id": "D7lJqW2PRFcN"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "DATASET=\"<YOUR-DATASET-HERE>\"\n",
+        "TEMP_LOCATION=\"<YOUR-TEMP-LOCATION-HERE>\"\n",
+        "PROJECT='<YOUR-PROJECT-ID-HERE>'\n",
+        "LOCATION='us-central1'\n",
+        "URL=f'https://healthcare.googleapis.com/v1/projects/{PROJECT}/locations/{LOCATION}/services/nlp:analyzeEntities'\n",
+        "NLP_SERVICE=f'projects/{PROJECT}/locations/{LOCATION}/services/nlp'\n",
+        "GCS_BUCKET=PROJECT"
+      ],
+      "metadata": {
+        "id": "s9lhe5CZ5F3o"
+      },
+      "execution_count": 5,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "**BigQuery Setup**\n",
+        "\n",
+        "We will be using BigQuery to warehouse the structured data revealed in the output of the Healthcare NLP API. For this purpose, we create 3 tables to organize the data. Specifically, these will be table entities, table relations, and table entity mentions, which are all outputs of interest from the Healthcare NLP API."
+      ],
+      "metadata": {
+        "id": "DI_Qkyn75LO-"
+      }
+    },
+    {
+      "cell_type": "code",
```

Comment:
Should there be a cell prior to this one to prompt the user to login? I ran this cell and naturally received an error of not being authenticated.